modules/web-app-manifest/vue/plugins/Manifest.server.ts

```diff
@@ -2,7 +2,7 @@ import type { App } from 'vue'
 import webAppManifest from '~/.sfx/webAppManifest'
 
 export const after = async (app: App, ctx?: any) => {
-  let content = `<link rel="manifest" href="/manifest.webmanifest">`
+  let content = `<link rel="manifest" crossorigin="use-credentials" href="/manifest.webmanifest">`
```

Comment:
This does not seem like a proper fix. It was not present before and it worked. Maybe look into the server route returning the web manifest JSON?